.travis.yml

```diff
@@ -162,21 +164,18 @@ matrix:
       stage: main-test
     - os: linux
       jdk: openjdk8
-      dist: trusty
```

Comment:
why remove this in the context of this PR?

Reply:
Travis does not work with dotnet if this dist is used, see this build for example https://travis-ci.org/github/antlr/antlr4/builds/751735906

Does it matter?